store/datadog.go

```diff
@@ -0,0 +1,20 @@
+package store
+
+import (
+	"sync"
+
+	"database/sql/driver"
+
+	"github.com/replicatedcom/saaskit/tracing/datadog"
+)
+
+// Datadog driver is not threadsafe and can panic during startup
+var ddLock = sync.Mutex{}
+
+func RegisterDatadogDriver(driverName string, driver driver.Driver, dbName string) {
```

Comment:
If this function is only called every once, highly recommend using `sync.Once` rather than a mutex.

Reply:
It's called once per database